app/assets/stylesheets/components/_featured_trips.scss

```diff
@@ -21,47 +11,158 @@
   flex-wrap: wrap;
   justify-content: center;
   align-items: center;
+  gap: 1vh;
 }
 
 .featured-card-img-top {
-  border-top-left-radius: 10px;
-  border-top-right-radius: 10px;
+  border-radius: 10px;
   object-fit: cover;
-  height: 60%;
-  width: 100%;
+  height: 80vw;
+  width: 80vw;
+}
+// link that surrounds card
+.feature-link {
+  text-decoration: none;
+  color: $black;
+  &:hover {
+    color: $lighter-black;
+    h3 {
+      color: $lighter-black;
+    }
+  }
 }
-
 .featured-card {
   border-radius: 10px;
-  box-shadow: 0px 0px 5px darkgrey;
+  // box-shadow: $box-shadow;
   display: flex;
   flex-direction: column;
   justify-content: center;
   align-items: center;
   height: 100%;
-  width: 30%;
+  width: 100%;
   overflow: hidden;
-  margin: 1.5%;
+  transition: 0.3s ease;
+  &:hover {
+    // box-shadow: $box-shadow-hover-card;
+    transition: 0.3s ease;
+  }
 }
 
+// things to do
+// style trip type
+
+
+
+
 .featured-card-body {
-  height: 20vh;
+  width: 80vw;
   font-family: $body-font;
-  text-decoration: none;
+  a {
+    text-decoration: none;
+  }
   display: flex;
   flex-direction: column;
 }
 
 .featured-body-title {
-  color:$gray;
-  font-size: 1.5rem;
-  padding: 0.8rem;
-  text-align: center;
+  color:$black;
+  font-size: 1.2rem;
+  padding-top: 0.8rem;
+  padding-bottom: 0.1rem;
+  margin: 0;
+  white-space: nowrap;
 }
+.featured-body-type {
+  color:$black;
+  font-size: 1rem;
+  margin-bottom: 0;
+  white-space: nowrap;
+  height: min-content;
+  margin-top: 0.3rem;
+}
+
+// trip type bottom right
+// title always on one line
 
 .featured-card-text {
-  padding: 1.5rem;
-  p {
-    height: min-content;
+  display: flex;
+  align-items: baseline;
+  justify-content: space-between;
+  h5 {
+    color: $gray;
+    font-size: 0.8rem;
   }
 }
+
+.mountains {
+  display: none;
+}
```

Comment:
On mobile version of the site, mountain featured trips will not display. This is because on mobile, featured trips are 100% wide. As such, it takes a while to scan through them all. To solve this, less featured trips appear.